instrumentation/aerospike-client/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.aerospike")
+    module.set("aerospike-client")
+    versions.set("[7.1.0,)")
+    assertInverse.set(true)
+  }
+}
+
+val latestDepTest = findProperty("testLatestDeps") as Boolean
+
+dependencies {
+  if (latestDepTest) {
+    library("com.aerospike:aerospike-client:+")
+  } else {
+    library("com.aerospike:aerospike-client:7.1.0")
+  }
```

Comment:
just `library("com.aerospike:aerospike-client:7.1.0")` would be enough, when latest dep tests are run version for `library` dependencies is automatically changed to latest